python/cuml/dask/common/part_utils.py

```diff
@@ -143,7 +145,7 @@ def _extract_partitions(dask_obj, client=None):
         parts = client.compute([p for p in zip(*parts)])
 
     else:
-        raise TypeError("Unsupported dask_obj type: " + type(dask_obj))
+        raise TypeError("Unsupported dask_obj type: " + str(type(dask_obj)))
```

Comment:
It would be nice to see a string formatting used like `“Unsupported dask_obj type: %s” % type(dask_obj)`. Would lower the possibility for little logic bugs like this.